src/xma/src/xmaapi/xma_utils.cpp

```diff
@@ -64,8 +64,8 @@ namespace xma_core {
         { ert_cmd_state::ERT_CMD_STATE_MAX, "MAX"}
     };
 
-    std::string get_cu_cmd_state(ert_start_kernel_cmd *cu_cmd) {
-        auto it = cu_cmdMap.find((ert_cmd_state)cu_cmd->state);
+    std::string get_cu_cmd_state(ert_cmd_state state) {
```

Comment:
What is this function trying to do? looks like a strange function :-(

Reply:
@chvamshi-xilinx ,This function return the state of CU in string format which is used for logging purpose.

Reply:
What about adding a comment before every function describing what it does?

Reply:
@keryell , added comments before every function.